webpack.config.js

```diff
@@ -31,12 +31,20 @@ module.exports = (env, { mode }) => ({
 	resolve: {
 		extensions: [".js", ".jsx"],
 		mainFields: ["browser", "main", "module"],
+		fallback: {
+			fs: false,
+		},
 	},
 	plugins: [
 		new webpack.NormalModuleReplacementPlugin(/^node:/u, resource => {
 			resource.request = resource.request.replace(/^node:/u, "");
 		}),
-		new NodePolyfillPlugin(),
+		new NodePolyfillPlugin({
+			additionalAliases: ["process"],
+		}),
+		new webpack.DefinePlugin({
+			"process.versions": JSON.stringify({ node: "16.0.0" }),
+		}),
```

Comment:
Why do we need this?

Reply:
Without this, I am getting the following error:

`Uncaught TypeError: process.versions.node is undefined`